herddb-cli/pom.xml

```diff
@@ -59,31 +59,6 @@
     </dependencies>
     <build>
         <plugins>
-            <plugin>
```

Comment:
I think that if you drop this plugin the .jar won't be able to work.

We still need a uber jar, but we can remove anything that is not required to run the CLI.
I think that most of the weight is due to the rocksdb jar.
I would drop useless jars, like we did with the "thin" JDBC driver

Reply:
The shaded jar generate various problems with his shaded dependencies.
You can just download the services and use the cli with it or in a more complex project run it with your other libraries.
Shading every dependency has many classloading problems when you want to upgrade some dependency (and then for some "strange" behavior you find loaded old classes from the shaded lib)

Reply:
The Uber jar is not intended to be used as dependency but only to run the CLI from the command line.

I agree that we can let users use the zip file.
Also, now that we have docker images, it is super easy to run docker run -it .....